pandas/core/frame.py

```diff
@@ -1390,7 +1390,8 @@ def to_csv(self, path_or_buf=None, sep=",", na_rep='', float_format=None,
     def to_excel(self, excel_writer, sheet_name='Sheet1', na_rep='',
                  float_format=None, columns=None, header=True, index=True,
                  index_label=None, startrow=0, startcol=0, engine=None,
-                 merge_cells=True, encoding=None, inf_rep='inf', verbose=True):
+                 merge_cells=True, encoding=None, inf_rep='inf', verbose=True,
+                 freeze_panes=None):
```

Comment:
Add `freeze_panes` to the docstring here, which this function uses
https://github.com/pandas-dev/pandas/blob/9ddba8dc811184c49d9bc4df5dc97d505345ec23/pandas/core/generic.py#L1021